src/test/java/com/hydratereminder/command/CommandInvokerTest.java

```diff
@@ -14,8 +14,7 @@
 
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.BDDMockito.given;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
```

Comment:
Please try to not use wildcard imports.

https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
Thanks for the tip issue resolved